4-WebApp-your-API/4-1-MyOrg/README.md

```diff
@@ -1,225 +1,269 @@
 ---
-page_type: sample
-languages:
-  - csharp
-products:
-  - aspnet-core
-  - azure-active-directory  
-name: How to secure a Web API built with ASP.NET Core using the Microsoft identity platform (formerly Azure Active Directory for developers)
-urlFragment: active-directory-aspnetcore-webapp-openidconnect-v2
-description: "This sample demonstrates a ASP.NET Core Web App calling a ASP.NET Core Web API that is secured using Azure AD"
```

Comment:
the tags you removed power the code samples browser -> [https://docs.microsoft.com/en-us/samples/browse/?source=docs](https://docs.microsoft.com/en-us/samples/browse/?source=docs)

You will need to restore them.

Reply:
added